openstudiocore/CMakeLists.txt

```diff
@@ -1361,8 +1417,8 @@ set(all_exe_targets
   #ResultsViewer
 )
 
-if(NOT Qt5Static)
-  list(APPEND all_exe_targets "OpenStudio")
+if(BUILD_OS_APP)
+  list(APPEND all_exe_targets "OpenStudioApp")
```

Comment:
ditto. why do we need these all_foo_targets? I've overlooked this for a while, but it has been on my mind to review what all it might be doing.